.github/workflows/ci.yml

```diff
@@ -126,16 +110,7 @@ jobs:
 
       - name: Configure unit
         run: |
-          ./configure \
-            --prefix=${{ steps.dir.outputs.prefix }} \
-            --sbindir=${{ steps.dir.outputs.bin }} \
-            --logdir=${{ steps.dir.outputs.var }}/log \
-            --log=${{ steps.dir.outputs.var }}/log/unit/unit.log \
-            --runstatedir=${{ steps.dir.outputs.var }}/run \
-            --pid=${{ steps.dir.outputs.var }}/run/unit/unit.pid \
-            --control=unix:${{ steps.dir.outputs.var }}/run/unit/control.sock \
-            --modules=${{ steps.dir.outputs.prefix }}/lib/unit/modules \
-            --statedir=${{ steps.dir.outputs.var }}/state/unit \
+          sudo ./configure \
             --tests \
             --openssl \
             --njs \
```

Comment:
Right, Unit doesn't have to be installed to run the pytests...

Reply:
I think these have defaults as well in `/usr` yes?

Reply:
Yes, `/usr/local`